src/Pckg/Generic/View/dispatcherNotifications.vue

```diff
@@ -1,8 +1,14 @@
 <template>
     <div class="pckg-dispatcher-notifications-wrapper">
-        <div class="alert alert-dismissible alert-fixxed" role="alert"
-             :class="'alert-' + notification.type"
-             v-for="notification in notifications">
+        <div 
+            class="alert alert-dismissible alert-fixxed animated" role="alert"
+            :class="[ 
```

Comment:
@nemanjaglumac Looking great. Can you also:
 - remove .alert-fixxed (it was used before alerts were fixed to current position)
 - refactor :class attribute condition to component method as in iconClass method (to make html readable) 
 Thanks!

Reply:
@schtr4jh Thank you for valuable input.

- b247e3086da16562998e07475c6f51321d923300: this should do the work.